libnetwork/drivers/bridge/setup_ipv6_linux.go

```diff
@@ -39,14 +41,20 @@ func setupBridgeIPv6(config *networkConfiguration, i *bridgeInterface) error {
 	}
 
 	// Setting route to global IPv6 subnet
+	// TODO(robmry) - remove this? The bridge is 'down' at this point so I think it
+	//  always fails, and the route is added anyway when the bridge is set 'up'.
 	log.G(context.TODO()).Debugf("Adding route to IPv6 network %s via device %s", config.AddressIPv6.String(), config.BridgeName)
 	err = i.nlh.RouteAdd(&netlink.Route{
```

Comment:
Why are we making this netlink call here? I know we want to make all drivers responsible for setting up interfaces, etc... instead of going through some weird indirection, but it seems that's the only route this driver sets. Do you know why?

Alternatively, could we make sure `setupBridgeIPv6` is never called before the bridge iface is up?